localdev/docker-compose.yml

```diff
@@ -26,7 +27,9 @@ services:
       - PGPORT=5432
       - POSTGRES_PASSWORD=P0stG&es
       - POSTGRES_DB=trino_gateway_db
+      - PGDATABASE=trino_gateway_db
       - POSTGRES_USER=trino_gateway_db_admin
+      - PGUSER=trino_gateway_db_admin
```

Comment:
The postgres docker image uses POSTGRES_DB and POSTGRES_USER instead of PGDATABASE and PGUSER https://hub.docker.com/_/postgres. We shouldn't need to specify the same value in two different ways.

Reply:
So if we remove this .. then we get the error message that @Chaho12 mentioned in the description.. so we must be doing something else wrong..

Reply:
We can remove this, and change https://github.com/trinodb/trino-gateway/pull/53/files#diff-51be12c366011b89d122169876c596658140cb407629a3eafac89827fcb3480cL33 to 
` ["CMD-SHELL","pg_isready -U $$POSTGRES_USER -d $$POSTGRES_DB || exit 1"]` (notice the `$$` instead of `$`)

Reply:
That would be better imho .. might need to add a comment though

Reply:
Fixed and added comment :)